native-embedding/src/test/c/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+cmake_minimum_required(VERSION 3.22)
```

Comment:
This file is a bit hard to read. Maybe add a newline before each of the comments.